x-pack/plugins/apm/kibana.json

```diff
@@ -11,7 +11,8 @@
     "embeddable",
     "infra",
     "observability",
-    "ruleRegistry"
+    "ruleRegistry",
+    "fleet"
```

Comment:
Can you please put this in alphabetical order?

Reply:
Addressed in https://github.com/elastic/kibana/pull/103554/commits/34f14653107c3e612c48b37029f00c4df1bad35a - I also sorted the other keys in this file. Let me know if that's an overstep and I'll revert.